ruma-api-macros/src/api/response.rs

```diff
@@ -220,6 +222,10 @@ impl TryFrom<RawResponse> for Response {
             ));
         }
 
+        // Header fields have to be emitted first to avoid use of moved value `response`
+        // compiler errors.
+        fields.sort();
```

Comment:
How is this only an issue with headers? Have you verified the codegen works for responses with multiple header fields?

Reply:
Actually the better way to impl Ord would be to sort `NewtypeRawBody` last as the `response.into()` is what causes problems.

An example of multi header is https://github.com/ruma/ruma/blob/master/ruma-client-api/src/r0/media/get_content.rs

Reply:
I see, so this isn't about headers at all. Makes sense that any newtype body fields would consume the request / response.